docs/site/src/demos/svg-icon/svg-icon.md

```diff
@@ -0,0 +1,20 @@
+# SVG Icon
```

Comment:
I think that having an Icons section under the `Style` section of the docs would be better.
https://material.google.com/style/icons.html#icons-system-icons